README.md

```diff
@@ -35,7 +35,17 @@ encrypted with which key, making it easy to age out keys when needed
 Keys are managed through a keyring--a short JSON document containing
 information about your encryption keys. The keyring must be a JSON
 array of objects with the fields `id`, `created_at`, and `value`. A
-keyring must have at least one key.
+keyring must have at least one key. For example:
+
+```json
+[
+  {
+    "id": "1380e471-038e-459a-801d-10e7988ee6a3",
```

Comment:
So the general idea is one would be obliged to choose an ever-higher ID and it's both to identify on rows what key in the keyring to use, as well as act as an ordering?

I think that would simplify things.